utils/vulcan.py

```diff
@@ -4,15 +4,14 @@
 from utils.modules_ext import *
 from utils.helper import *
 
-def RunVULCAN( atm, time_dict, loop_counter, dirs, runtime_helpfile, COUPLER_options ):
+def RunVULCAN( atm, time, loop_counter, dirs, runtime_helpfile, COUPLER_options ):
```

Comment:
About this simplification of the time dict: is it still possible to define a simulation relative to star formation? i.e., can one set up a simulation at 150 Myr after the formation of the Sun? Will this be displayede in the plots then as starting from 0, or starting from 150 My onwards?

Reply:
The stellar age is now also stored in the helpfile, so this should be relatively easy to do now, yes. It would simply be a case of plotting values vs `age_stellar` rather than versus `Time`.